js/customscripts.js

```diff
@@ -87,30 +87,59 @@ $(document).ready(function () {
 
 /*
 * Script to show/hide the buttons under the navigation tiles.
+* REPLACED BY THE SCRIPT BELOW.
 */
-window.addEventListener('DOMContentLoaded', function () {
-    document.querySelectorAll('.button').forEach(function (item) {
-        // Show the tiles when the cursor hovers over the <h2 class="button">.
-        // Does not apply to touchscreen devices. The CSS just shows all the
-        // subelements by default for those devices.
-        item.addEventListener('mouseenter', function () {
-            item.classList.add("active-button");
-            item.nextElementSibling.classList.add("active-child-box");
-        });
-        // Hide the child boxes when the cursor leaves the wrapper <div>
-        item.parentElement.addEventListener('mouseleave', function () {
-            item.classList.remove("active-button");
-            item.nextElementSibling.classList.remove("active-child-box");
-        });
-    });
-});
+// window.addEventListener('DOMContentLoaded', function () {
+//     document.querySelectorAll('.button').forEach(function (item) {
+//         // Show the tiles when the cursor hovers over the <h2 class="button">.
+//         // Does not apply to touchscreen devices. The CSS just shows all the
+//         // subelements by default for those devices.
+//         item.addEventListener('mouseenter', function () {
+//             item.classList.add("active-button");
+//             item.nextElementSibling.classList.add("active-child-box");
+//         });
+//         // Hide the child boxes when the cursor leaves the wrapper <div>
+//         item.parentElement.addEventListener('mouseleave', function () {
+//             item.classList.remove("active-button");
+//             item.nextElementSibling.classList.remove("active-child-box");
+//         });
+//     });
+// });
+//
+// function external_new_window() {
+//     for (var c = document.getElementsByTagName("a"), a = 0; a < c.length; a++) {
+//         var b = c[a];
+//         if (b.getAttribute("href") && b.hostname !== location.hostname) {
+//             b.target = "_blank";
+//             b.rel = "noopener";
+//         }
+//     }
+// }
 
-function external_new_window() {
```

Comment:
This script opens external links in a new tab, do we need to delete this?

Reply:
I guess so - I don't know anything about it. I saw it was causing errors in the console, but didn't want to get rid of it in case it was important. I meant to mention it to you.

Reply:
@martin-nc I updated the javascript pieces in #337 and it gives no errors in the console, so maybe we keep it and see what it gives after I've merged the carousel?